base-image-import.yml

```diff
@@ -131,3 +149,13 @@ jobs:
       scriptPath: tag-new-image.sh
       arguments: |
         "$(baseTag)" "$(targetImage)" "$(acrName)" "$(targetRegistry)" "$(baseDigest)" "$(acrDigest)" 
+
+  - task: AzureCLI@1
```

Comment:
do we need a separate one for failed? to call it out better?

Reply:
It's  not that simple to implement for what I can see, so I skipped it for now as we have other priorities